src/GovUk.Education.ExploreEducationStatistics.Admin/Services/LoggingNotificationClient.cs

```diff
@@ -23,7 +22,9 @@ public LoggingNotificationClient(ILogger<LoggingNotificationClient> logger)
             Dictionary<string, dynamic> personalisation = null,
             string clientReference = null, string emailReplyToId = null)
         {
-            _logger.Log(LogLevel.Debug, $"Sending email to {emailAddress} with template id {templateId}");
+            _logger.Log(LogLevel.Debug, 
+                "Sending email to {EmailAddress} with template id {TemplateId}", 
```

Comment:
I'm not sure we should be logging the email address anywhere even at this log level. You could probably remove this log message.